src/components/BooksList.js

```diff
@@ -0,0 +1,26 @@
+import PropTypes from 'prop-types';
+
+const Bookslist = ({ books, handleDelete }) => (
+  <div className="booksList">
+    {books.map((book) => (
+      <div className="prevBooks" key={book.id}>
+        {book.title}
```

Comment:
- Render a component of type `Book` here. A component allows for reusability of code across the app. Please make that change.